ElevatorSim/src/elevatorsim/floor/Floor.java

```diff
@@ -1,57 +1,59 @@
 package elevatorsim.floor;
 
 import java.util.HashMap;
-import java.util.Set;
+import java.util.List;
 
-import elevatorsim.common.MessageReciever;
 import elevatorsim.common.MessageRequest;
 import elevatorsim.enums.Direction;
 
-public class Floor implements MessageReciever {
+/**
+ * 
+ *
+ */
+public class Floor {
```

Comment:
This should still implement MessageReciever. Synchronizer needs that to send the floor data.

Reply:
What are your thoughts on having FloorController implement MessageReciever instead? If the scheduler wont end up calling any of the floor methods directly, maybe we may not need to store all the various instances of type Floor in the scheduler.

Reply:
Oh yeah! I forgot we made that change